### ISSUE TITLE
Solve SVM problem

### ISSUE THREAD

**Issue by ikopeykin** [ikopeykin/IBA#1]
In this task you will train SVM models for the problem.

1. In new Jupyter notebook create CountVectorizer and vectorize the dataset.
2. Randomly split dataset to training and validation sets using train_test_split(). Hold out 15% of dataset for validation. Use stratification by labels (flags) to keep the same balance in training and validation sets.
3. Train SVC() model for each flag type. Try different values for C (0.1, 1.0, 10.0).
4. Evaluate each model by ROC AUC on training and validation set. Calculate mean ROC AUC.
5. Push your notebook to Gerrit for review.
